packaging/makespec.sh

```diff
@@ -24,3 +24,20 @@ echo "# Auto-generated from $(basename $RPMSPEC_IN) by makespec.sh" | cat - $RPM
 sed -i -e "s/@api_version@/$API_VERSION/g" $RPMSPEC
 sed -i -e "s/@rpm_version@/$RPM_VERSION/g" $RPMSPEC
 sed -i -e "s/@nuget_version@/$NUGET_VERSION/g" $RPMSPEC
+
```

Comment:
I think @WonyoungChoi wants rewritting this part to a separate python script and calling it here instead installing `jq`.
Am I correct?

Reply:
Created a separate Python script and modified it to call it.